src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1515,10 +1514,6 @@ private uint Run(string bannerText, string helpText, bool isPrestartWarned, stri
 
                 _runspaceRef.Runspace.Close();
                 _runspaceRef = null;
-                if (staMode) // don't recycle the Runspace in STA mode
-                {
-                    ShouldEndSession = true;
```

Comment:
If I understand correctly - we default to STA and remove the option to use MTA, correct?

Assuming so, this line should probably stay, but I don't quite understand why it's here.

Reply:
No, we defaulted to MTA and removing option to use STA